requirements.txt

```diff
@@ -58,6 +58,7 @@ pytest-cov == 2.10.0
 pytest-django == 3.8.0
 factory-boy == 3.0.0
 pytest-factoryboy == 2.0.3
+faker
```

Comment:
Holder ikke `factory`?

Reply:
Hva brukes denne til? Bør vel også spesifisere en kompatibel versjon så ikke denne oppdaterer seg og de andre ikke gjør det

Reply:
er faker som i gidd tenkte egentlig bare å snike den inn for gøy @Eirsteir, kan ta den vekk bruke sbare i en test